persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAO.kt

```diff
@@ -120,7 +120,7 @@ data class ProposalTimerEntity(
 )
 
 interface ConversationDAO {
-    suspend fun getSelfConversationId(): QualifiedIDEntity?
+    suspend fun getSelfConversationId(protocol: ConversationEntity.Protocol): QualifiedIDEntity?
```

Comment:
shouldn't we set a default value here or update everywhere we use it? I thing this might break lot of places.

Reply:
It's not used in many places as you can see in this PR. The UI shouldn't interact directly with the self conversation.

I rather keep it explicit because we will soon start switching between MLS and proteus self conversation depending on if MLS is enabled or not.